akka-http-caching/src/main/scala/akka/http/caching/LfuCache.scala

```diff
@@ -111,7 +112,19 @@ private[caching] class LfuCache[K, V](val store: AsyncLoadingCache[K, V]) extend
 
   def apply(key: K, genValue: () ⇒ Future[V]): Future[V] = store.get(key, toJavaMappingFunction[K, V](genValue)).toScala
 
-  def getOrLoad(key: K, loadValue: K ⇒ Future[V]) = store.get(key, toJavaMappingFunction[K, V](loadValue)).toScala
+  def getOrLoad(key: K, loadValue: K ⇒ Future[V]): Future[V] =
+    store.get(key, toJavaMappingFunction[K, V](loadValue)).toScala
+
+  def put(key: K, mayBeValue: Future[V])(implicit ex: ExecutionContext): Future[Unit] = {
```

Comment:
Actually, since `put` doesn't return anything anyway, perhaps we should return `Unit` here and leave it up to the caller to decide whether or not to attach error handling to the `Future[V]`? Or just return the `Future[V]` to make it more explicit that it's the users' responsibility to do error handling?

Reply:
I think returning Future[V] will better